build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -1961,9 +1961,30 @@ message Compressor {
 
     // Brotli compression.
     BROTLI = 3;
+
+    // Zstandard compression with dictionary.
```

Comment:
Is the dictionary optional?

If a server supports this mode, should we also require that it support ZSTD (ie without dictionary)?

Reply:
Currently, I prefer to think of this as a separate compressor and not related to ZSTD at all. It helps simplify the change I am proposing. Using a mix of ZSTD and ZSTD_DICT is a non-goal for me at this moment.